labs/fabex.md

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+Fabex
+
+# Short Description
+The aim of this lab is to create a flexible block explorer for Hyperledger Fabric written in Go.
```

Comment:
I would rephrase it a bit, how about?

"The aim of this library is to provide microservice which augments and index the ledger of Hyperledger Fabric and to provide additional set of complex queries APIs to extract provenance information out of the ledger."

Reply:
I think, it's right.